packages/rolldown/src/utils/bindingify-output-options.ts

```diff
@@ -70,6 +70,7 @@ export function bindingifyOutputOptions(
     sanitizeFileName,
     preserveModules,
     virtualDirname,
+    comments: outputOptions.comments,
```

Comment:
NIT:
you could destruct `comments` from `outputOptions` just like other fields

Reply:
DONE